tests/sniffTest

```diff
@@ -68,15 +68,13 @@ if [[ $rc != 0 || -s err ]]; then
 	echo Output: $(cat out)
 	exit 1
 fi
-$RUNC kill c1
-rm -rf /run/opencontainer/containers/c1
```

Comment:
What's the issue with trying to stop and remove the container here? I'd prefer if each test cleaned-up after itself.